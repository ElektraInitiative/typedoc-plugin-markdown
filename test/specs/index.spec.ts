import * as Handlebars from 'handlebars';
import { ProjectReflection } from 'typedoc';

describe(`Index:`, () => {
  let project: ProjectReflection;
  let indexTemplate: Handlebars.TemplateDelegate;

  beforeAll(async () => {
    project = await global.bootstrap(['reflections.ts']);
    indexTemplate = global.getTemplate('index');
    global.stubHelpers(['breadcrumbs']);
  });

  test(`should compile readme`, () => {
    expect(
      global.compileTemplate(indexTemplate, {
        model: project,
        project: project,
      }),
    ).toMatchSnapshot();
  });
});
