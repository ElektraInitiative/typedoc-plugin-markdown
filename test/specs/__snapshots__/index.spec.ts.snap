// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index: should compile readme 1`] = `
"[helper: breadcrumbs]

# MY README

> Some block quote

<!-- some comments -->

Some text
"
`;
